setup.cfg

```diff
@@ -90,6 +90,7 @@ docs =
 	jaraco.packaging >= 9
 	rst.linker >= 1.9
 	jaraco.tidelift >= 1.4
+	sphinx-notfound-page == 0.8.3
```

Comment:
I rebased the commit to fix the conflict (and decided to leave the version pinned because the extension haven't achieved 1.0 yet).

Reply:
Sounds good! It has been pretty stable and we don't have plans for big changes under the radar --but I think it's fine to use the pinned version as a way of self protection just in case 😄